### ISSUE TITLE
Support includes

### ISSUE THREAD

**Issue by janoszen** [fralau/mkdocs_macros_plugin#7]
When trying to use Jinja2 includes the following happens:

```
ERROR   -  Error reading page '***MASKED**.md': no loader for this environment specified
[E 190417 16:34:02 ioloop:909] Exception in callback <bound method LiveReloadHandler.poll_tasks of <class 'livereload.handlers.LiveReloadHandler'>>
    Traceback (most recent call last):
      File "***MASKED***\venv\lib\site-packages\tornado\ioloop.py", line 907, in _run
        return self.callback()
      File "***MASKED***\venv\lib\site-packages\livereload\handlers.py", line 66, in poll_tasks
        filepath, delay = cls.watcher.examine()
      File "***MASKED***\venv\lib\site-packages\livereload\watcher.py", line 93, in examine
        func()
      File "***MASKED***\venv\lib\site-packages\mkdocs\commands\serve.py", line 114, in builder
        build(config, live_server=live_server, dirty=dirty)
      File "***MASKED***\venv\lib\site-packages\mkdocs\commands\build.py", line 274, in build
        _populate_page(file.page, config, files, dirty)
      File "***MASKED***\venv\lib\site-packages\mkdocs\commands\build.py", line 174, in _populate_page
        'page_markdown', page.markdown, page=page, config=config, files=files
      File "***MASKED***\venv\lib\site-packages\mkdocs\plugins.py", line 94, in run_event
        result = method(item, **kwargs)
      File "***MASKED***\venv\lib\site-packages\macros\plugin.py", line 71, in on_page_markdown
        return md_template.render(**self.variables)
      File "***MASKED***\venv\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
        return original_render(self, *args, **kwargs)
      File "***MASKED***\venv\lib\site-packages\jinja2\environment.py", line 1008, in render
        return self.environment.handle_exception(exc_info, True)
      File "***MASKED***\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
        reraise(exc_type, exc_value, tb)
      File "***MASKED***\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
        raise value.with_traceback(tb)
      File "<template>", line 235, in top-level template code
    TypeError: no loader for this environment specified
```

The expected behavior is that the specified filename should be interpreted relative to the currently processed file and the specified file should be loaded. The resulting content should be processed as markdown.

**Comment by fralau** [comment]
Thanks. Could you give a sample of what you wrote in the markdown?

**Comment by janoszen** [comment]
Something along these lines:

```
{% include “otherfile.j2” %}
```

**Comment by fralau** [comment]
I have not investigated in detail yet. Most likely, the jinja template would use the directory in which the plugin is installed, as the default location. If you want to use `otherfile.j2`, could you try to use an absolute path?

**Comment by janoszen** [comment]
I did. The exception sounds to me like jinja2 needs some sort of path
resolver to know where to load the templates from. It would make sense
since the templates could be coming from a DB too.

**Comment by Justin-W** [comment]
@fralau FYI: PR#9 solves my use case. Thanks to @mlenkeit for submitting it.

I'm now installing from his fork until PR#9 gets merged into an official release, since `import` is an essential feature for us to be able to use jinja effectively within our mkdocs site.

**Comment by ryandaugherty** [comment]
Yes!!! This fork works like a charm for includes now. Thanks for submitting @mlenkeit

**Comment by fralau** [comment]
This is solved with 28c2ce8
Thanks @mlenkeit
